### PR TITLE
feat(admin-ui): Add Arabic translations

### PR DESCRIPTION
**Pull request by PeterWadie**

### CODE REVIEW

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 1]
```diff
@@ -0,0 +1,1919 @@
+{
+  "back-button-go-back": "رجوع",
+  "sales-channels-display-available-count": "متاح في <2>{{availableChannelsCount}}</2> من <6>{{totalChannelsCount}}</6> قنوات المبيعات",
+  "activity-drawer-activity": "النشاط",
+  "activity-drawer-no-notifications-title": "هادئ هنا...",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "activity-drawer-no-notifications-title": "هدوء هنا...",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 2]
```diff
@@ -0,0 +1,1919 @@
...
+  "activity-drawer-error-title": "آه لا...",
+  "activity-drawer-error-description": "حدث خطأ ما أثناء محاولة جلب إشعاراتك - سنواصل المحاولة!",
+  "activity-drawer-processing": "جار المعالجة...",
+  "analytics-config-form-title": "جعل بيانات الاستخدام الخاصة بي مجهولة الهوية",
+  "analytics-config-form-description": "يمكنك اختيار جعل بيانات استخدامك مجهولة الهوية. إذا تم تحديد هذا الخيار ، فلن نقوم بجمع معلوماتك الشخصية ، مثل اسمك وعنوان بريدك الإلكتروني.",
+  "analytics-config-form-opt-out": "عدم مشاركة بيانات الاستخدام الخاصة بي",
+  "analytics-config-form-opt-out-later": "يمكنك دائمًا اختيار عدم مشاركة بيانات الاستخدام الخاصة بك في أي وقت.",
+  "analytics-preferences-success": "نجاح",
+  "analytics-preferences-your-preferences-were-successfully-updated": "تم تحديث تفضيلاتك بنجاح",
+  "analytics-preferences-error": "خطأ",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "analytics-preferences-error": "حدث خطأ",
```

I think it makes sense to use this translation for all "Error" translations

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 3]
```diff
@@ -0,0 +1,1919 @@
...
+  "analytics-preferences-help-us-get-better": "ساعدنا في التحسن",
+  "analytics-preferences-disclaimer": "لإنشاء أكثر تجربة تجارة إلكترونية شيقة، نود الحصول على رؤى حول كيفية استخدامك لـ Medusa. تسمح لنا رؤى المستخدمين ببناء منتج أفضل وأكثر جاذبية وأكثر قابلية للاستخدام. نحن نجمع البيانات فقط لتحسين المنتجات. اقرأ البيانات التي نجمعها في",
+  "analytics-preferences-documentation": "الوثائق",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "analytics-preferences-documentation": "وثائقنا",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 4]
```diff
@@ -0,0 +1,1919 @@
...
+  "analytics-preferences-please-enter-a-valid-email": "الرجاء إدخال بريد إلكتروني صالح",
+  "analytics-preferences-continue": "استمر",
+  "currency-input-currency": "العملة",
+  "currency-input-amount-is-not-valid": "المبلغ غير صالح",
+  "organisms-success": "نجاح",
+  "organisms-delete-successful": "تم الحذف بنجاح",
+  "organisms-are-you-sure-you-want-to-delete": "هل أنت متأكد أنك تريد الحذف؟",
+  "organisms-no-cancel": "لا، إلغاء",
+  "organisms-yes-remove": "نعم، إزالة",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "organisms-yes-remove": "نعم، إحذف",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 5]
```diff
@@ -0,0 +1,1919 @@
...
+  "error-boundary-back-to-dashboard": "العودة إلى لوحة التحكم",
+  "error-boundary-an-unknown-error-occured": "حدث خطأ غير معروف",
+  "error-boundary-bad-request": "طلب خاطئ",
+  "error-boundary-you-are-not-logged-in": "أنت غير مسجل الدخول",
+  "error-boundary-you-do-not-have-permission-perform-this-action": "ليس لديك إذن لتنفيذ هذا الإجراء",
+  "error-boundary-page-was-not-found": "لم يتم العثور على الصفحة",
+  "error-boundary-an-unknown-server-error-occured": "حدث خطأ خادم غير معروف",
+  "error-boundary-503": "الخادم غير متاح حاليًا",
+  "error-boundary-500": "حدث خطأ بأسباب غير محددة، ومن المرجح أن يرجع ذلك إلى مشكلة فنية من جانبنا. يرجى تحديث الصفحة. إذا استمرت المشكلة، يرجى الاتصال بالمسؤول الخاص بك.",
+  "error-boundary-400": "كان طلب غير صحيح، قم بإصلاح طلبك ثم حاول مرة أخرى.",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "error-boundary-400": "الطلب غير صحيح، قم بإصلاح طلبك ثم حاول مرة أخرى.",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 6]
```diff
@@ -0,0 +1,1919 @@
...
+  "error-boundary-401": "أنت غير مسجل الدخول ، يرجى تسجيل الدخول للمتابعة.",
+  "error-boundary-403": "ليس لديك إذن لتنفيذ هذا الإجراء ، إذا كنت تعتقد أن هذا خطأ ، اتصل بالمسؤول الخاص بك.",
+  "error-boundary-404": "لم يتم العثور على الصفحة التي طلبتها ، يرجى التحقق من عنوان URL وحاول مرة أخرى.",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "error-boundary-404": "لم يتم العثور على الصفحة التي طلبتها ، يرجى التحقق من عنوان الصفحة والمحاولة مرة أخرى.",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 7]
```diff
@@ -0,0 +1,1919 @@
...
+  "gift-card-denominations-section-and-more_one": ", و {{count}} أكثر",
+  "gift-card-denominations-section-and-more_other": ", و {{count}} أكثر",
+  "gift-card-denominations-section-delete-denomination": "حذف الفئة",
+  "gift-card-denominations-section-confirm-delete": "هل أنت متأكد أنك تريد حذف هذه الفئة؟",
+  "gift-card-denominations-section-denomination-deleted": "تم حذف الفئة",
+  "gift-card-denominations-section-denomination-was-successfully-deleted": "تم حذف الفئة بنجاح",
+  "gift-card-denominations-section-edit": "تعديل",
+  "gift-card-denominations-section-delete": "حذف",
+  "help-dialog-how-can-we-help": "كيف يمكننا مساعدتك؟",
+  "help-dialog-we-usually-respond-in-a-few-hours": "عادةً ما نرد خلال بضع ساعات",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "help-dialog-we-usually-respond-in-a-few-hours": "عادةً نرد خلال بضع ساعات",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 8]
```diff
@@ -0,0 +1,1919 @@
...
+  "help-dialog-subject": "الموضوع",
+  "help-dialog-what-is-it-about": "ماذا يدور حوله؟...",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "help-dialog-what-is-it-about": "عن ماذا الموضوع؟...",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 9]
```diff
@@ -0,0 +1,1919 @@
...
+  "help-dialog-feel-free-to-join-our-community-of": "لا تتردد في الانضمام إلى مجتمعنا من",
+  "help-dialog-merchants-and-e-commerce-developers": "التجار ومطوري التجارة الإلكترونية",
+  "help-dialog-send-a-message": "إرسال رسالة",
+  "invite-modal-success": "نجاح",
+  "invite-modal-invitation-sent-to": "تم إرسال الدعوة إلى {{user}}",
+  "invite-modal-error": "خطأ",
+  "invite-modal-member": "عضو",
+  "invite-modal-admin": "مدير",
+  "invite-modal-developer": "مطور",
+  "invite-modal-invite-users": "دعوة مستخدمين",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "invite-modal-invite-users": "إدعو مستخدمين",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 10]
```diff
@@ -0,0 +1,1919 @@
...
+  "invite-modal-invite": "دعوة",
+  "login-card-no-match": "هذه البيانات غير مطابقة لسجلاتنا.",
+  "login-card-log-in-to-medusa": "تسجيل الدخول إلى Medusa",
+  "login-card-email": "البريد الإلكتروني",
+  "login-card-password": "كلمة المرور",
+  "login-card-forgot-your-password": "هل نسيت كلمة المرور؟",
+  "metadata-add-metadata": "إضافة بيانات وصفية",
+  "product-attributes-section-edit-attributes": "تحرير السمات",
+  "product-attributes-section-dimensions": "الأبعاد",
+  "product-attributes-section-configure-to-calculate-the-most-accurate-shipping-rates": "قم بالتهيئة لحساب أكثر معدلات الشحن دقة",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "product-attributes-section-configure-to-calculate-the-most-accurate-shipping-rates": "قم بالتهيئة لحساب معدلات الشحن بأكثر دقة",
```

File: packages/admin-ui/ui/public/locales/ar/translation.json [hunk 11]
```diff
@@ -0,0 +1,1919 @@
...
+  "collection-modal-successfully-created-collection": "تم إنشاء المجموعة بنجاح",
+  "collection-modal-edit-collection": "تعديل المجموعة",
+  "collection-modal-add-collection": "إضافة مجموعة",
+  "collection-modal-description": "لإنشاء مجموعة ، كل ما تحتاجه هو عنوان ومعالج.",
+  "collection-modal-details": "التفاصيل",
+  "collection-modal-title-label": "العنوان",
+  "collection-modal-title-placeholder": "النظارات الشمسية",
+  "collection-modal-handle-label": "المُعالج",
+  "collection-modal-handle-placeholder": "النظارات الشمسية",
+  "collection-modal-slug-description": "عنوان URL للمجموعة. سيتم إنشاؤه تلقائيًا إذا ترك فارغًا.",
```
Comment (shahednasser) [inline packages/admin-ui/ui/public/locales/ar/translation.json]:
```suggestion
  "collection-modal-slug-description": "عنوان الصفحة للمجموعة. سيتم إنشاؤه تلقائيًا إذا ترك فارغًا.",
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution! I think the translation is much better than last time. I left some suggestions for fixes.

Can you also run the command `yarn changeset` in the root of the monorepository, choose the `admin-ui` package, and set the message to the title of this PR?

**Review by shahednasser — APPROVED** [review]
LGTM!

@olivermrbl would you like to sign off on this as well?

**Comment by olivermrbl** [comment]
@shahednasser, any specific reason this introduces a major bump? I think we are fine with a patch no?

**Comment by shahednasser** [comment]
no reason, I think I just missed that 😅

**Pull request merged**